components/CardContainer/CardContainer.styled.js

```diff
@@ -1,4 +1,4 @@
-import { calculateFontSize } from "@/utils";
+import { calculateFontSize } from "../../utils";
```

Comment:
this is a really neat trick! you can also look into a css property named clamp. it lets you define a minimum and maximum font size and a responsive one in the middle.

Reply:
thanks for the good address!   😃
diese unterschiedlichen Importvorzeichen entstehen immer je nachdem, wer sie anlegt. Scheinbar handhaben unsere vs code-Programme das unterschiedlich. Sehr nervig, aber gut gesehen. Ich dachte, ich hätte alle erwischt ;)